docker/docker.sh

```diff
@@ -2,6 +2,10 @@
 
 set -euxo pipefail
 
+readonly NOT_SUPPORTED_MESSAGE="Docker initialization action is not supported on Dataproc 2.0+.
```

Comment:
It is more accurate to replace 2.0+ with `${DATAPROC_VERSION}`.

Reply:
Done.